bindings/matlab/iDynTreeMATLAB_wrap.cxx

```diff
@@ -1,6 +1,6 @@
 /* ----------------------------------------------------------------------------
  * This file was automatically generated by SWIG (http://www.swig.org).
- * Version 3.0.9
+ * Version 3.0.5
```

Comment:
With which commit did you generate the bindings? https://github.com/jaeandersson/swig/tree/5d3770e452b71265162001c2249dea7961660629 should be 3.0.9 .